src/Underscore.php

```diff
@@ -83,9 +119,13 @@ public function __call($method, $args)
      */
     public static function __callStatic($method, $args)
     {
-        $payloadClass = sprintf('\Underscore\Initializer\%sInitializer', ucfirst($method));
-        /** @var $payload callable */
-        $payload = new $payloadClass();
+        $payload = static::fromMixin($method);
```

Comment:
I'm not entirely sure about using mixins for initializers... does it make sense to have that functionality? Or should it only exist for mutators?